### DISCUSSION TITLE
Github should also show which framework or library is using in the project

### DISCUSSION THREAD

**Discussion by weberankit** [community/community#59653] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

When I first started coding and tried to contribute to projects on GitHub, I encountered many difficulties because many projects use frameworks or libraries that are only mentioned in their project documentation. For instance, if anyone knows javascript and if anyone has to contribute to the project on GitHub and filters the project based on language, it will show a javascript project but that will not tell whether that project uses react or angular so it is very difficult to tell whether that project uses react or angular.

I believe the GitHub project language section should display the framework and library used in the project
![image](https://github.com/community/community/assets/94105514/e46e266c-46cf-43b9-b28a-8f68d3633db0)

**Comment by weberankit** [comment]
your suggestion or advice is welcome

**Comment by DSDmark** [comment]
Hi @weberankit,

It showed dependencies. But, not on the front page of the repo. If you want to see it. Then, please click on **Insights**. There you find everything about your repo like this:

![image](https://github.com/community/community/assets/85744816/27b71168-18db-4d58-90e2-df7139468c5b)

Here you can see details about your **repo**.

![image](https://github.com/community/community/assets/85744816/a300cf6c-0e68-49a2-b634-b305cc9c7ef8)

---

If you want to make that feature request to GitHub, that is show dependencies in front page of the repo. Then, change your label into **Product feedback** and make your title something like that: 

> `Feature Request: GitHub should show framework or library in repo front page.`

This well caught the attention of GitHub members and maintainers.
